x-pack/plugins/security_solution/public/management/pages/policy/view/policy_forms/components/policy_form_layout.tsx

```diff
@@ -153,16 +154,18 @@ export const PolicyFormLayout = React.memo(() => {
       <EuiBottomBar paddingSize="s">
         <EuiFlexGroup justifyContent="flexEnd" gutterSize="s">
           <EuiFlexItem grow={false}>
-            <EuiButtonEmpty
-              color="ghost"
-              onClick={handleCancelOnClick}
-              data-test-subj="policyDetailsCancelButton"
-            >
-              <FormattedMessage
-                id="xpack.securitySolution.endpoint.policy.details.cancel"
-                defaultMessage="Cancel"
-              />
-            </EuiButtonEmpty>
+            <EuiThemeProvider colorMode="dark">
+              <EuiButtonEmpty
+                color="text"
```

Comment:
Why is ThemeProvider needed here?

Also, this should still use `ghost` for `color` if you're using the theme provider. Else with this change, I see this:

![Screenshot 2022-11-04 at 14 10 49](https://user-images.githubusercontent.com/1849116/199980466-6a8a563d-11d2-43fe-afb1-af4074f61993.png)

(notice the faded empty button)

instead of this, where the empty button text has way more contrast.
![Screenshot 2022-11-04 at 14 11 04](https://user-images.githubusercontent.com/1849116/199980554-5fe9df6e-e36b-4868-9201-0641dfb6954c.png)

Reply:
@ashokaditya this is because the `ghost` color was deprecated with the recent `EuiButton` conversion. In order to use this color now, you should wrap the button in an `EuiThemeProvider` with the dark color mode and use the color `text`. I've place the PR it came from below for reference.

I also have a question about your screenshot because when I got the Policies page tog see that bottom bar, I'm not seeing a faded version. I've left a screenshot to what I'm seeing below. Would you example be in a different spot?

> Updated button ghost colors to be colorMode = 'dark' themed text buttons (https://github.com/elastic/eui/pull/6150)

<img width="1370" alt="image" src="https://user-images.githubusercontent.com/40739624/199990578-e97b0d71-1bd8-4e37-90a3-a00639fab82b.png">

Reply:
> @ashokaditya this is because the `ghost` color was deprecated with the recent `EuiButton` conversion. In order to use this color now, you should wrap the button in an `EuiThemeProvider` with the dark color mode and use the color `text`. I've place the PR it came from below for reference.
I see. 
> I also have a question about your screenshot because when I got the Policies page tog see that bottom bar, I'm not seeing a faded version. I've left a screenshot to what I'm seeing below. Would you example be in a different spot?

My bad. I was running a stale kibana instance. I rechecked this and it renders as expected. So this is all good. 👍